app/src/debug/AndroidManifest.xml

```diff
@@ -29,35 +28,6 @@
 
     <!-- *** Fastlane permissions end *** -->
 
-    <!-- *** ADM permissions (temporarily in debug only) *** -->
-    <!-- This permission ensures that no other application can intercept your ADM messages. -->
-    <permission
-        android:name="${applicationId}.permission.RECEIVE_ADM_MESSAGE"
-        android:protectionLevel="signature"/>
-    <uses-permission android:name="${applicationId}.permission.RECEIVE_ADM_MESSAGE" />
-    <uses-permission android:name="com.amazon.device.messaging.permission.RECEIVE" />
-    <!-- This permission is also brought in by androidx.work:work-runtime dependency. -->
-    <uses-permission android:name="android.permission.WAKE_LOCK"/>
-    <!-- *** ADM permissions end *** -->
-
     <application>
```

Comment:
We should be able to remove the application tag now. I added that in with the ADM permissions.